R/classInt.R

```diff
@@ -358,6 +358,7 @@ classIntervals <- function(var, n, style="quantile", rtimes=3, ..., intervalClos
         dots <- list(...)
         iqr_mult <- ifelse(is.null(dots$iqr_mult), 1.5, dots$iqr_mult)
         qtype <- ifelse(is.null(dots$type), 7, dots$type)
```

Comment:
A small comment here, should be enforced a minimum 0 value in order to prevent errors?  

```r
iqr_mult <- max(0, ifelse(is.null(dots$iqr_mult), 1.5, dots$iqr_mult))
```

With negative values we get again the error:

``` r
packageVersion("classInt")
#> [1] '0.4.9'

set.seed(101)

x <- rnorm(50)

classInt::classIntervals(x, style = "box")
#> style: box
#>   one of 1,906,884 possible partitions of this variable into 6 classes
#>         [-Inf,-2.618441)   [-2.618441,-0.7198015) [-0.7198015,-0.02713441) 
#>                        0                       13                       12 
#>  [-0.02713441,0.5459584)     [0.5459584,2.444598)           [2.444598,Inf] 
#>                       12                       13                        0

classInt::classIntervals(x, style = "box", iqr_mult = -0.01)
#> style: box
#>   one of 1,906,884 possible partitions of this variable into 6 classes
#> Error in findInterval(clI$var, clI$brks, all.inside = TRUE): 'vec' must be sorted non-decreasingly and not contain NAs

# No error on 0
classInt::classIntervals(x, style = "box", iqr_mult = max(0, -0.01))
#> style: box
#>   one of 1,906,884 possible partitions of this variable into 6 classes
#>   [-2.319327,-0.7198015)  [-0.7198015,-0.7198015) [-0.7198015,-0.02713441) 
#>                       13                        0                       12 
#>  [-0.02713441,0.5459584)    [0.5459584,0.5459584)     [0.5459584,1.427756] 
#>                       12                        0                       13
```

<sup>Created on 2023-02-22 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>